doc/source/contrib/stats.rst.mako

```diff
@@ -0,0 +1,69 @@
+.. _contrib-stats:
+
+Statistics
+----------
+
+The statistics functionality allows you to gather usage information within pyramid_oereb
+itself. The data gathered will be persisted in configured database,
+allowing the operator or owner of the application to query at any time
+the usage statistics for that database.
+
+The functionality is configured via the server's ini file; see the project repository for a
+complete example of such an ini file.
```

Comment:
@danduk82 we need to add here a description of the "args", for example a list, explaining for each element what it can be used for.
Can you please make a proposal?